webpack.config.js

```diff
@@ -40,4 +40,5 @@ const genConfig = (filename, { inlineCss } = {}) => ({
 module.exports = [
   genConfig('index-without-styles.js'),
   genConfig('index.js', { inlineCss: true }),
+  genConfig('button-react.js'),
```

Comment:
This is an output file name, have you specified the input one? The built `button-react.js` is equal to `index-without-styles.js`, it doesn't contain a React version of Superhero button.

Reply:
Got that

Reply:
Done